### PR TITLE
[PL48459] SMP 0.14.5 Patch Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
https://harness.atlassian.net/browse/PL-48459

Added two tickets for the patch of SMP 0.14.5

https://harness.atlassian.net/browse/PL-48373
https://harness.atlassian.net/browse/CDS-94037

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: release-notes/self-managed-enterprise-edition.md [hunk 1]
```diff
@@ -67,6 +67,50 @@ If you don't use Helm to upgrade Harness Self-Managed Enterprise Edition, follow
...
+  "gs://smp-airgap-bundles/harness-0.14.5/ci_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.14.5/ff_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.14.5/platform_images.tgz" \
+  "gs://smp-airgap-bundles/harness-0.14.5/sto_images.tgz" \
+  .
+```
+
+### New features and enhancements
+
+- The Environments page and the Infrastructure Definition section of the Environments page didn't list more than 100 infrastructure definitions. This issue is resolved by adding pagination and search support. You can now view 500 items in infrastructure selection drop-down in the Run Pipeline page. (CDS-94037)
```
Comment (roshnisarangadharan) [inline release-notes/self-managed-enterprise-edition.md:108]:
```suggestion
- The Environments page and the Infrastructure Definition section of the Environments page didn't list more than 100 infrastructure definitions. This issue is resolved by adding pagination and search support. You can now view 500 items in the infrastructure selection drop-down in the Run Pipeline page. (CDS-94037)
```

**Comment by saifeemustafaqharness** [comment]
@pratmit @nataraja2018 @bharatgoelharness Need review and approval to merge this. SMP 0.14.5 has already been released

**Review by pratmit — APPROVED** [review]
LGTM

**Review by brian-f-harness — APPROVED** [review]
LGTM. Please check to be sure that there are no Zendesk (ZD tickets) we need to reference for the items.

**Pull request merged**